widget/app/src/constants.ts

```diff
@@ -1 +1,5 @@
 export const WC_PROJECT_ID = 'e24844c5deb5193c1c14840a7af6a40b';
+export const TREZOR_MANIFEST = {
+  appUrl: 'https://rango.exchange/',
+  email: 'hi@rango.exchange',
```

Comment:
```suggestion
  email: 'hi+trezorwidget@rango.exchange',
```

Reply:
Shouldn't we put the correct email address and URL for the app?

Reply:
We should pass `trezorManifest` in `app-v2` repo in a separate repo.